### DISCUSSION TITLE
Installation issue with Monterey 12.0.1

### DISCUSSION THREAD

**Discussion by Mailisyorlik** [jsvine/pdfplumber#561] [Q&A]
Hello all, 
I am trying to create a PDF to dataframe reader/writer using pdfplumber, but I am running into installation issues of some kind. 
I get the following error thrown when trying to run my program:

```
ImportError: MagickWand shared library not found.
You probably had not installed ImageMagick library.
Try to install:
  brew install freetype imagemagick
```

I have tried what feels like every trick in the book for reinstalling/updating/reordering my directories to try to fix this. I have also tried to install some of the libraries that pdfplumber depends on piecemeal but it does not change anything.
What am I missing? 

Thank you in advance for any help!

**Comment by jsvine** [comment]
Does following these steps resolve the situation?: https://stackoverflow.com/questions/37011291/python-wand-image-is-not-recognized/41772062#41772062

Cf. https://github.com/jsvine/pdfplumber/issues/81

**Reply by Mailisyorlik** [reply]
So when I run that symlink command, it just says 

ln: ./libMagickWand.dylib: File exists

which is weird because it seems like my python file can't find it

**Reply by samkit-jain** [reply]
Do these help? https://docs.wand-py.org/en/0.6.7/guide/install.html#install-imagemagick-on-mac and https://docs.wand-py.org/en/0.6.7/guide/install.html#explicitly-link-to-specific-imagemagick

**Reply by Mailisyorlik** [reply]
still does not seem to want to behave. Getting the same error as before where it cannot find the MagickWand library. Should I do a fresh reinstall in some other directory or something?

**Reply by Mailisyorlik** [reply]
should I try moving the whole thing to a venv?
